CleanCode.SolidPrinciples/OpenClosed/Observer.cs

```diff
@@ -4,38 +4,15 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace CleanCode.SolidPrinciples
+namespace CleanCode.SolidPrinciples.OpenClosed
 {
     public class Observer
     {
-        public bool Observe(ObservationPlace place)
+        public bool Observe(IPlace place)
         {
-            string[] ignoredSightings;
-            string result;
-
-            switch (place.Type)
-            {
-                case PlaceType.Forest:
-                    ignoredSightings = new[] { "|", "@" };
-                    Console.WriteLine("<Forest Sounds>");
-
-                    result = place.Environment;
-                    foreach (var ignored in ignoredSightings)
-                        result = result.Replace(ignored, string.Empty);
-
-                    return result.Length > 0;
-                case PlaceType.Ocean:
-                    ignoredSightings = new[] { "~" };
-                    Console.WriteLine("bul bul bul");
-
-                    result = place.Environment;
-                    foreach (var ignored in ignoredSightings)
-                        result = result.Replace(ignored, string.Empty);
-
-                    return result.Length > 0;
-                default:
-                    throw new InvalidOperationException("Unknown place");
-            }
+          
+            return place.checkLength();
```

Comment:
Schowanie wszystkiego w checkLength() powoduje, że z tego poziomu nie wiadomo co właściwie się dzieje. Może warto to podzielić na osobne funkcje wypisania czegoś na ekran i osobną do ogarnięcia tych ignoredSightings?